src/eventmaster/eventmaster.go

```diff
@@ -72,6 +72,13 @@ func getDbConfig() dbConfig {
 	return dbConf
 }
 
+func setupPlugins() {
+	if _, err := os.Stat("plugins/github.json"); err == nil {
```

Comment:
eventmaster should have its own config file, a hardcoded config for the plugin isn't good-- nor is it good to infer that you should run with just that stat (esp. since the file might not be there on startup).

If eventmaster itself doesn't have config-- it needs it :)